src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 50;
+  const liteDiscount = 20;
+  const pricePerDay = 40;
+  let totalPrice = 0;
+
+  totalPrice += pricePerDay * days;
```

Comment:
You may set expression as a value of the variable
```suggestion
  const totalPrice = pricePerDay * days;
```